bec_lib/bec_lib/devicemanager.py

```diff
@@ -650,16 +650,16 @@ def _add_device(self, dev: dict, msg: DeviceInfoMessage):
         class_name = info["device_info"]["device_class"]
 
         if base_class == "device":
-            logger.info(f"Adding new device {name}")
+            logger.debug(f"Adding new device {name}")
```

Comment:
I'm not sure about the change of log level. On one hand, it greatly reduces the spamming. On the other hand, it can be very useful whilst trying to see some progress when devices are slow to connect. But maybe we remove it for now and if we really can't live without it, we add it again? Or we replace the direct logs with just a summary of devices that have been added?